.github/workflows/ci.yml

```diff
@@ -16,7 +16,7 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  test:
+  test_targeted:
```

Comment:
Shouldn't this be `test` then instead? In the `ANALYSIS` matrix you specify the different test runs, right?